metadata-service/auth-impl/src/main/java/com/datahub/authorization/DataHubAuthorizer.java

```diff
@@ -151,6 +152,7 @@ public AuthorizedActors authorizedActors(
       // Step 3: For each matching policy, add actors that are authorized.
       authorizedUsers.addAll(matchingActors.getUsers());
       authorizedGroups.addAll(matchingActors.getGroups());
+      authorizedRoles.add(matchingActors.getRoles());
```

Comment:
Something doesn't look right here, is authorizedRoles a list of lists? Should this be addAll?